cmd/configtxgen/main.go

```diff
@@ -41,7 +49,7 @@ func doOutputBlock(config *genesisconfig.Profile, channelID string, outputBlock
 		return errors.New("refusing to generate block which is missing orderer section")
 	}
 	if config.Consortiums != nil {
-		logger.Info("Creating system channel genesis block")
+		logger.Error("'Consortiums' must be nil since system channel is no longer supported")
```

Comment:
The message should be 
"Warning: 'Consortiums' should be nil since system channel is no longer supported in Fabric v3.x"